scss/civicrm/administer/civi-case/_case-types.scss

```diff
@@ -1,3 +1,32 @@
 &.page-civicrm-a {
   @include civihr_table();
+
+  .crmCaseType {
+
+    .crm-button {
+      @extend %btn-civi;
+
+      &[crm-icon=fa-check] {
+        @extend %btn-civi-primary;
+        padding: 8px 19px !important;
+      }
+
+      &[crm-icon=fa-times] {
+        @extend %btn-civi-secondary-outline;
+        color: $gray !important;
+        border: solid 1px $gray-dark !important;
+        padding: 7px 19px !important;
+        margin: 0;
+
+        &:hover {
+          background: $gray-dark;
+          color: $crm-white !important;
+        }
+      }
+    }
+
+    .crm-block {
```

Comment:
I see this block padding being fixed multiple times, shouldn't we have a generic css rule that fixes it once and for all?